### PR TITLE
Runs on my machine now

### PR DESCRIPTION
**Pull request by bennett000**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -10,4 +10,12 @@ const server = createServer({ config, dbClients});
 
 server.listen(config.port, () => {
   console.log(`listening on ${config.port}`);
+});
+
+process.on('uncaughtException', (err: Error) => {
```
Comment (darrenmce) [inline src/index.ts]: eek ur swallowing exceptions lol, I have a module for doing this kinda stuff that I'll dig up

Comment (bennett000) [reply src/index.ts]: That's actually noise... I was trying to figure something else out when I added those

File: src/config/index.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const defaultConfig: OAuthConfig = {
   },
   dbs: {
     redis: {
-      host: 'localhost',
+      host: 'redis',
```
Comment (darrenmce) [inline src/config/index.ts]: shouldn't need to change this here, docker-compose has the env variable to override this, and if local u can use the ignored dot file (rc)

Comment (bennett000) [reply src/config/index.ts]: ah I need to make that ignored dot file.  Is there a template for `.oauth-serverrc`?

File: docker-compose.yml [hunk 3]
```diff
@@ -7,8 +7,15 @@ services:
       oauth-server_stores__redis__host: redis
     links:
       - redis
+    depends_on:
```
Comment (darrenmce) [inline docker-compose.yml]: I think depends_on is just a subset of links, is there a diff u know of?

Comment (bennett000) [reply docker-compose.yml]: this was probably reactionary to `EADDRBLOCKED` errors the redis connector was throwing

File: src/config/index.ts (hunk 2 above)
Comment (darrenmce) [reply src/config/index.ts]: nah it just mimics the config object, whatever u want to override (don't need the whole structure)

File: docker-compose.yml (hunk 3 above)
Comment (bennett000) [reply docker-compose.yml]: (which a config file will solve)

**Review by darrenmce — APPROVED** [review]
LGTM, nothing to hold it up.. merge when ur happy

**Comment by bennett000** [comment]
@darrenmce what kind of merging/basing do you want?

(I tried the rebase option but "Oops Something Went Wrong", permissions?)

**Comment by darrenmce** [comment]
squash and merge is fine, I don't like github UI Reba sing.. scary

**Pull request merged**